verilog/analysis/verilog_linter.cc

```diff
@@ -332,10 +333,51 @@ absl::Status PrintRuleInfo(std::ostream* os,
   return absl::OkStatus();
 }
 
+static void AppendCitation(CustomCitationMap& citations,
+                           const absl::string_view& rule_raw) {
+  const size_t eq_pos = rule_raw.find(':');
+  if (!eq_pos || eq_pos == absl::string_view::npos) return;
```

Comment:
I'd explicitly compare `eq_pos == 0` because that is what we mean. `!eq_pos` evaluates to the same thing, but doesn't reflect what we mean.